### PR TITLE
nfreq_avg rename

### PR DESCRIPTION
**Pull request by mwilensky768**
This branch just renames nfreq_avg to beam_nfreq_avg everywhere it appears. I tested this on one keyword set where I set the old keyword  and another keyword set where I set the new keyword to something other than default. The behavior was appropriate in both cases. There were some minor issues during rebase, namely that I mistakenly changed some of the simulation wrappers in a previous commit, which were then deleted on master.

### CODE REVIEW

File: dictionary.md [hunk 1]
```diff
@@ -499,9 +499,12 @@ WARNING! Options in this section may change without notice, and should never be
   -*Default*: 0 <br />
   -*eor_wrapper_defaults*: 1 <br />
 
+<<<<<<< deafd82b048010b433a7baa7efb600dffe5e31c6
```
Comment (nicholebarry) [inline dictionary.md]: fix merge conflict

File: dictionary.md [hunk 2]
```diff
@@ -499,9 +499,12 @@ WARNING! Options in this section may change without notice, and should never be
...
 **obs_id**: the unique identifier for the observation. Examples are GPS seconds or Julian Dates. The input uvfits file must share this unique identifier name: `<obs_id>.uvfits`.    
   -*Default*: not set    
 
+=======
```
Comment (nicholebarry) [inline dictionary.md]: fix merge conflict

**Comment by mwilensky768** [comment]
The keyword has been added to `fhd_depreciation_test` in pipeline_scripts.

**Comment by nicholebarry** [comment]
Approved

**Pull request merged**